jboss_wildfly/datadog_checks/jboss_wildfly/data/conf.yaml.example

```diff
@@ -0,0 +1,238 @@
+  ## @param host - string - required
+  ## JBoss/Wildfly host to connect to
+  #
+  - host: localhost
+
+  ## @param port - integer - required
+  ## JBoss/Wildfly port to connect to
+  #
+    port: 7091
+
+  ## @param tags  - list of key:value element - optional
+  ## List of tags to attach to every metric, event and service check emitted by this integration.
+  ##
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - environment: default_environment
+
+  ## @param user - string - optional
+  ## JMX username
+  #
+  #  user: <USERNAME>
+
+  ## @param password - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  password: <PASSWORD>
+
+  ## @param process_name_regex - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  process_name_regex: .*process_name.*
+
+  ## @param tools_jar_path - string - optional
+  ## Instead of specifying a host, and port. The agent can connect using the attach api.
+  ## This requires the JDK to be installed and the path to tools.jar to be set below.
+  ## Note: It needs to be set when process_name_regex parameter is set
+  #
+  #  tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
+
+  ## @param name - string - optional
+  ## XXXXXXXXXXX
+  #
+  #   name: jboss_instance
+
+  ## @param java_bin_path - string - optional
+  ## java_bin_path should be set if the agent cannot find your java executable
+  #
+  #  java_bin_path: <JAVA_PATH>
+
+  ## @param java_options - string - optional
+  ## List of Java JVM options.
+  #
+  #  java_options: "-Xmx200m -Xms50m"
+
+  ## @param trust_store_path - string - optional
+  ## trust_store_path should be set if ssl is enabled.
+  ## path to your trusted store
+  #
+  #  trust_store_path: <TRUSTSTORE.JKS_PATH>
+
+  ## @param trust_store_password - string - optional
+  ## trust_store_password should be set if ssl is enabled
+  ## password for your TrustStore.jks file
+  #
+  #  trust_store_password: password
+
+  ## @param key_store_path - string - optional
+  ## key_store_path should be set if client authentication is enabled on the target JVM.
+  ## path to your key store
+  #
+  #  key_store_path: <KEYSTORE.JKS_PATH>
+
+  ## @param key_store_password - string - optional
+  ## key_store_password should be set if client authentication is enabled on the target JVM.
+  ## password for your KeyStore.jks file
+  #
+  #  key_store_password: <PASSWORD>
+
+  ## @param rmi_registry_ssl - boolean - optional
+  ## Whether or not the agent should connect to the rmi registry using ssl.
+  #
+  #  rmi_registry_ssl: false
+
+
+init_config:
+
+  ## @param is_jmx - boolean - required - default: true
+  ## Whether or not this file is a configuration for a JMX integration
+  #
+  is_jmx: true
+
+  ## @param collect_default_metrics - boolean - required - default: true
+  ## Whether or not the check should collect all default metrics for this integration.
+  #
+  collect_default_metrics: true
+
+  # Additional section to allow jmx connection to JBoss/Wildfly.
```

Comment:
This comments block needs to be consistent with the rest of the file.

Reply:
Done, thanks.